Makefile

```diff
@@ -461,7 +465,8 @@ derived_sources += $(drifted_files) Driver/Version.hs $(hs_der_files)
     count fromKif release cgi ghci build-hets callghc \
 	get-programatica check_desktop check_server check_cgi \
 	install install-common install-owl-tools archive \
-	build-indep build-arch build binary-indep binary-arch binary rev.txt
+	build-indep build-arch build binary-indep binary-arch binary rev.txt \
+	stack-setup stack-install-dependencies
```

Comment:
No. That's why I'm asking for help ;)

Oh, and I see that this is something I forgot to remove after I failed to invoke Stack as a dependent target at the right time and then moved to call it for a variable in the var.mk.